### ISSUE TITLE
better documentation of serializing CNMF models to disk and loading them in

### ISSUE THREAD

**Issue by kushalkolar** [flatironinstitute/CaImAn#1141]
We generally need better documentation of how to do this for both online and offline CNMF.

**Comment by EricThomson** [comment]
Yes this is the cause of many issues where people are trying to roll their own solutions and don't realize there are built-in methods already.

**Comment by EricThomson** [comment]
This is explained better in the recently updated demo notebook for cnmf (PR #1075 ). 

For online.... 😬 @kushalkolar does that work well out of the box or is it a nightmare? I need to push online stuff up my queue.

**Comment by kushalkolar** [comment]
I think that currently anyone who's using OnACID is an advanced programmer so they'll figure it out somehow.
